hi-ml/src/health_ml/utils/config_loader.py

```diff
@@ -0,0 +1,152 @@
+#  ------------------------------------------------------------------------------------------
+#  Copyright (c) Microsoft Corporation. All rights reserved.
+#  Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+#  ------------------------------------------------------------------------------------------
+from __future__ import annotations
+
+import importlib
+import inspect
+import logging
+import sys
+from importlib.util import find_spec
+from pathlib import Path
+from typing import Any, Dict, List, Optional
+
+import param
+from importlib._bootstrap import ModuleSpec
+
+from health_ml.lightning_container import LightningContainer
+from health_ml.utils.common_utils import path_to_namespace
+
+
+class ModelConfigLoader(param.Parameterized):
+    """
+    Helper class to manage model config loading
+    """
+    model_configs_namespace: Optional[str] = param.String(default=None,
+                                                          doc="Non-default namespace to search for model configs")
```

Comment:
Consider movinkg the model_config_namespace argument to ExperimentConfig, and make that a plain class. Not sure why we had that odd construct in IE-DL, maybe because we had more fields for the loader at some point.

Reply:
As per the comment above, have removed the model_config_namespace argument and replaced with fully qualified model names